documentation/examples/remote_storage/remote_storage_bridge/README.md

```diff
@@ -0,0 +1,62 @@
+# Remote storage bridge
+
+This is a bridge that receives samples in Prometheus's remote storage
+format and forwards them to Graphite, InfluxDB, or OpenTSDB. It is meant
+as a replacement for the built-in specific remote storage implementations
+that will be removed from Prometheus soon.
```

Comment:
Ok. If we remove them in 1.6 (which would be great), then I could work on that right afterwards and also reintroduce multiple remote writers already.